session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -913,6 +914,27 @@ public SessionDataSet executeQueryStatement(String sql)
         execResp.isIgnoreTimeStamp());
   }
 
+  public SessionDataSet executeRawDataQuery(List<String> paths, long startTime, long endTime)
```

Comment:
add comments,  like the query is select * from paths where time >= startTime and time < endTime, time interval include startTime and exclude endTime

Reply:
and change all other "time" in the code to the constant string, like in BasicOperatorType